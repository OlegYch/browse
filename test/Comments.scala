class Comments {
	
	// A single line comment
	val x = 3

	/* A plain comment */
	def y(c: List[Double]): Int = ???

	/** A doc comment
	* @param t A parameter
	*/
	def z[T](t: T): T = t

}