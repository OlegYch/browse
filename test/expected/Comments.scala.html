<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8" ></meta>
        <title>Comments.scala</title>
        <script type="text/javascript" src="jquery-all.js"></script>
        <script type="text/javascript" src="linked.js"></script>
        <link rel="stylesheet" type="text/css" href="style.css" title="Style"></link>
    </head>
    <body>
        <pre>
<span class="keyword">class</span> <a title="class Comments extends AnyRef" id="6936">Comments</a> <a href="#6936" title="Comments" class="delimiter">{</a>
	
	<span class="comment">// A single line comment</span>
	<span class="keyword">val</span> <a title="Int" id="16317">x</a> = <span title="Int(3)" class="int">3</span>

	<span class="comment">/* A plain comment */</span>
	<span class="keyword">def</span> <a title="(c: List[Double])Int" id="16319">y</a><span class="delimiter">(</span><a title="List[Double]" id="16324">c</a>: <span title="List[Double]">List</span><span class="delimiter">[</span>Double<span class="delimiter">]</span><span class="delimiter">)</span>: <span title="Int">Int</span> = <span title="=&gt; Nothing">???</span>

	<span class="comment">/** A doc comment
	* @param t A parameter
	*/</span>
	<span class="keyword">def</span> <a title="[T](t: T)T" id="16320">z</a><span class="delimiter">[</span><a title="" id="16322">T</a><span class="delimiter">]</span><span class="delimiter">(</span><a title="T" id="16325">t</a>: <a href="#16322" title="T">T</a><span class="delimiter">)</span>: <a href="#16322" title="T">T</a> = <a href="#16325" title="T">t</a>

<span class="delimiter">}</span>
        </pre>
    </body>
</html>
